config/webpack/webpack.common.js

```diff
@@ -146,8 +146,4 @@ const webpackConfig = {
     },
 };
 
-if (platform === 'desktop') {
-    webpackConfig.target = 'electron-renderer';
```

Comment:
This is no longer needed – because we're using context isolation, the renderer process (our app that's bundled by webpack) is just like a regular web app without any direct access to Electron or Node APIs, except those defined in the `contextBridge`